Tests.Integration.Presentation.Web/Organizations/StsOrganizationSynchronizationApiTest.cs

```diff
@@ -546,6 +546,255 @@ public async Task Can_PUT_UPDATE_Consequences_With_Conversion_Consequences()
             Assert.Null(convertedUnit.ExternalOriginUuid);
         }
 
+        [Fact]
+        public async Task Can_GET_LOGS_With_Addition_Consequences()
+        {
+            //Arrange
+            var cookie = await HttpApi.GetCookieAsync(OrganizationRole.GlobalAdmin);
+            var targetOrgUuid = await CreateOrgWithCvr(AuthorizedCvr);
+            const int firstRequestLevels = 2;
+            const int secondRequestLevels = 3;
+            using var postResponse = await SendPostCreateConnectionAsync(targetOrgUuid, cookie, firstRequestLevels);
+
+            //Act
+            using var consequencesResponse = await SendGetUpdateConsequencesAsync(targetOrgUuid, secondRequestLevels, cookie);
+            Assert.Equal(HttpStatusCode.OK, consequencesResponse.StatusCode);
+            var consequences = await consequencesResponse.ReadResponseBodyAsKitosApiResponseAsync<ConnectionUpdateConsequencesResponseDTO>();
+
+            using var putResponse = await SendPutUpdateConsequencesAsync(targetOrgUuid, secondRequestLevels, cookie);
+            Assert.Equal(HttpStatusCode.OK, putResponse.StatusCode);
+
+            using var logsResponse = await SendGetLogsAsync(targetOrgUuid, 5, cookie);
```

Comment:
only this line is part of the ACT... the other two are prerequisites thereby making them part of the "arrange"